### PR TITLE
Created new file ShivamPathak99.md and updated Data.json

### PR DESCRIPTION
**Pull request by ShivamPathak99**
## Description

Added ShivamPathak99.md in journeys and updated Data.json .

*Please consider it for Hacktoberfest*

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by Adarsh-jaiss — APPROVED** [review]

**Comment by Adarsh-jaiss** [comment]
- @vikasganiga05  please fix this merge conflict!

**Review by vikasganiga05 — APPROVED** [review]
It looks good to me. Thanks for your contribution :tada:
- Thank you so much Shivam Pathak for sharing your journey till now.
- We are waiting for more contributions from you :green_heart:

Hi @Adarsh-jaiss , I resolved the merge conflicts. please share your thoughts on the journey file.

File: journeys/ShivamPathak99.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<h1>Hi, this is Shivam Pathak☺️ </h1>
```
Comment (vikasganiga05) [inline journeys/ShivamPathak99.md:1]:
```suggestion
# [Shivam Pathak](https://github.com/ShivamPathak99)'s Journey
```

**Review by vikasganiga05 — CHANGES_REQUESTED** [review]
Hi @ShivamPathak99 , A small suggestion.

File: journeys/ShivamPathak99.md (hunk 1 above)
Comment (vikasganiga05) [reply journeys/ShivamPathak99.md:1]: Hey, no changes are made :thinking:

**Review by vikasganiga05 — APPROVED** [review]
It looks good to me. Thanks for your contribution :tada:
- Thank you so much Shivam Pathak for sharing your journey till now.
- We are waiting for more contributions from you :green_heart:

**Pull request merged**